packages/ucp-enforcement/src/util/Util.ts

```diff
@@ -0,0 +1,54 @@
+import { RequestInfo } from "../storage/ContainerUCRulesStorage";
+import { Store } from "n3";
+import { turtleStringToStore } from "./Conversion";
+
+
+export async function readLdpRDFResource(fetch: (input: RequestInfo, init?: RequestInit | undefined) => Promise<Response>, resourceURL: string): Promise<Store> {
```

Comment:
This funtion is only used in the ContainerUCRulesStorage, so I would suggest moving it there.